conda/recipes/libcuvs/build_libcuvs_tests.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 # Copyright (c) 2022-2024, NVIDIA CORPORATION.
 
-./build.sh tests --allgpuarch --no-nvtx --build-metrics=tests_bench --incl-cache-stats
+./build.sh tests bench-ann --allgpuarch --no-nvtx --build-metrics=tests_bench --incl-cache-stats
```

Comment:
Over in RAFT, `bench-ann` is its own package with its own dependencies, build scripts, etc.:

https://github.com/rapidsai/raft/tree/877644a423c0268746af62cecb7150afa65d8386/python/raft-ann-bench

https://github.com/rapidsai/raft/blob/877644a423c0268746af62cecb7150afa65d8386/conda/recipes/raft-ann-bench/meta.yaml

For my own understanding in reviewing this... why is this being added to the `libcuvs-tests` package here instead of creating a new standalone package as exists in RAFT?

Having these separate things be their own packages can be helpful for parallelizing development, limiting the potential impact of packaging changes, and speeding up debugging.

Reply:
I agree that benchmarks should probably be a separate package. I'm only hesitating to add this because I'm not very familiar with this conda/CI setup. I hoped to postpone this question till a follow-on PR together with the naming decision (https://github.com/rapidsai/cuvs/issues/160#issuecomment-2173310776). @benfred, @cjnolet , what do you think?